docs/weightonlyquant.md

```diff
@@ -40,6 +40,8 @@ Our motivation is improve CPU support for weight only quantization, since `bitsa
 
 ### LLM Runtime example code
 If `use_llm_runtime` is enabled, the LLM Runtime backend is used, the default value is True.

```

Comment:
shall we update LLM runtime to NeuralSpeed in all the places? We can still keep use_llm_runtime as legacy option to keep backward compatible, but our code/doc should change to use_neural_speed sth like that.